### ISSUE TITLE
Upgrade deprecated `resolve-url` library

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa-admin#1232]
**Describe the bug**
When the command `npm install --legacy-peer-deps` is run for the first time as part of the installation process, this error message is seen.

```
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated

```

**To Reproduce**
- See above

**Expected behavior**
1. No error message because the library has been updated, or suitably replaced, not because the error message has been suppressed.
1. All tests pass and are updated for the new library
1. If the package is no longer used, then it must be removed
1. No application functionality is affected

**Actual behavior**
- See above

**Screenshots**
- N/A

**Additional details**
- N/A

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by hharshas** [comment]
would like to work on this issue

**Comment by palisadoes** [comment]
@hharshas You have requested to be assigned to 10 issues in the space of 5 minutes. Everyone needs to get a chance to work on issues. Once you have been assigned two issues, the next person in line in the remaining 8 will be provided access. Please watch this video for guidance

- https://www.youtube.com/watch?v=3mjap16GYiY

Your actions are not in keeping with the spirit of the community. Please adjust your activities to be more collaborative.

@Cioppolo14 @noman2002 @Kevoniat please take note.

**Comment by hharshas** [comment]
@palisadoes would keep that in mind from now onwards

**Comment by Devesh326** [comment]
Hey, I would like to work on this issue!

**Comment by Vankydwivedi** [comment]
Hello @palisadoes  , This issue has been inactive for time being now and I would like to work on this issue. Can you assign this to me?

**Comment by Vankydwivedi** [comment]
Hey @palisadoes, while working on this issue, I discovered that jest-enzyme relies on resolve-url through a lengthy dependency chain. I attempted to address the issue by updating jest-enzyme, hoping it would resolve the problem. Unfortunately, the update didn't work as expected. Upon further investigation, I identified conflicting dependencies, such as enzyme-adapter-react-16 and airbnb-prop-types, both relying on an outdated version of react-16. Despite my efforts, I couldn't find a satisfactory solution. As this is my first encounter with such an issue, I would greatly appreciate some guidance. Thanks!

**Comment by palisadoes** [comment]
1. This complex interdependence has happened a few times in the past. This is probably why dependabot couldn't fix it automatically. 
2. We'll close this for now.
3. Please update all the dependencies with `npm audit fix`, make sure all the tests pass and submit a PR. Don't fix using the force flag that will cause breaking changes.

**Comment by Vankydwivedi** [comment]
I appreciate your guidance; I've understood the instructions and will work on it.

**Comment by Vankydwivedi** [comment]
Hello sir,
I am having difficulty in creating my PR.
The conditions are:

All checks should be passed.
Only submit PRs against the develop branch.
However, I am having difficulty in both.
One test is failing.
Another has been skipped.
When trying to merge my branch into the develop branch, it's saying 679 files are changed.
The test which was failing is in the third picture.
Screenshot 2024-01-25 020013

Screenshot 2024-01-31 185751

Screenshot 2024-01-30 232630

if you could help me a little it will be of great help for me to proceed,
thanks.

**Comment by palisadoes** [comment]
Please ask the talawa-admin channel in our slack community

**Comment by Vankydwivedi** [comment]
ok sir

**Comment by antriksh-9** [comment]
Hey @palisadoes I want to work on this issue.
Please assign it to me.

**Comment by antriksh-9** [comment]
Hey @palisadoes I think this issue has already been resolved by PR #1501 as in this PR `jest-enzyme` has been deleted and `resolve-url` depends on it. So running `npm install --leagcy-peer-deps` no more creates `npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated` this error.

Dependency tree of `resolve-url`:
![Screenshot 2024-02-08 143303](https://github.com/PalisadoesFoundation/talawa-admin/assets/132576984/014dae54-080a-4b67-871f-a3993176beb7)

After deleting `jest-enzyme` library: 
![Screenshot 2024-02-08 143647](https://github.com/PalisadoesFoundation/talawa-admin/assets/132576984/92608a4d-7f75-462f-ab84-b31dbbfcfdf7)

**Comment by antriksh-9** [comment]
Hey @palisadoes what about the other deprecated package warnings?
Do we need to upgrade/solve it?
